apps/web/src/views/Swap/components/AdvancedSwapDetails.tsx

```diff
@@ -178,6 +186,38 @@ export const TradeSummary = memo(function TradeSummary({
           <Text fontSize="14px">{`${formatAmount(realizedLPFee, 4)} ${inputAmount?.currency?.symbol}`}</Text>
         </RowBetween>
       )}
+
+      {isPaymasterAvailable && (
+        <>
+          <RowBetween style={{ padding: '4px 0 0 0' }}>
+            <RowFixed>
+              <Text fontSize="14px" color="textSubtle">
+                {t('Gas Token')}
+              </Text>
+              <QuestionHelper
+                text={
+                  <>
+                    <Text mb="12px">
+                      <Text bold display="inline-block">
+                        {t('Gas Token')}
+                      </Text>
+                      <br />
+                      <br />
+                      {t('Select a token to pay gas fees.')}
+                      <br /> <br />
+                      {t('Please refer to the transaction confirmation on your wallet for the final gas fee.')}
+                    </Text>
+                  </>
+                }
+                ml="4px"
+                placement="top"
+              />
+            </RowFixed>
+
+            {gasTokenSelector}
```

Comment:
I was thinking the whole row including the label is part of the gas token selector